pds4indextools/pds4_create_xml_index.py

```diff
@@ -359,9 +661,9 @@ def main():
                              'specified, all elements found in the XML files are '
                              'included.')
 
-    parser.add_argument('--xpaths', action='store_true',
-                        help='If specified, use full XPaths in the column '
-                             'headers. If not specified, use only elements tags.')
+    parser.add_argument('--simplify-xpaths', action='store_true',
+                        help='If specified, uses tags of unique XPaths. Any values with '
+                             'duplicate values will still use their full XPath.')
 
     parser.add_argument('--output-file', type=str,
                         help='The output filepath ending with your chosen filename for '
```

Comment:
Maybe my comment above isn't great:

Replace the ":" and "/" in the column headers with (legal friendly) characters can be made more specific: Replace ":" and "/" in column headers with underscores

since some are replaced with a single underscore, some with two, and some deleted entirely. Perhaps something like "Rename column headers such that they only contain characters permissible in variable names."